### ISSUE TITLE
what to log?

### ISSUE THREAD

**Issue by waleedsamy** [Ubicall/ubicall-web-service#76]
start add some end user details to be persisted, and begin with generic data about client such browser, IP, OS version..

this data will contain too info about top and lowest Api hitting, client who had exceeded Daily limit which now 5000 per day..

any now Ideas @hagry @amegalla ?

**Comment by waleedsamy** [comment]
fixed with #79

**Comment by waleedsamy** [comment]
@amegalla  @rawanhamdi  sample log documents
```javascript
{
    "_id": ObjectId("569cd8c5a2f7c9e856268cbb"),
    "url": "/call/queue/4/Information-Requests",
    "method": "GET",
    "params": {
        "queue_slug": "Information-Requests",
        "queue_id": "4"
    },
    "licence_key": "e6053eb8d35e02ae40beeeacef203c1a",
    "app_id": "ubicall-agent",
    "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.80 Safari/537.36",
    "status_code": 200,
    "status": "success",
    "datetime": ISODate("2016-01-18T12:21:25.585Z"),
    "category": "call",
    "__v": 0
} {
    "_id": ObjectId("569cd924a2f7c9e856268cbc"),
    "url": "/call/478/failed",
    "method": "PUT",
    "params": {
        "call_id": "478"
    },
    "licence_key": "e6053eb8d35e02ae40beeeacef203c1a",
    "app_id": "ubicall-agent",
    "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.80 Safari/537.36",
    "status_code": 200,
    "status": "success",
    "datetime": ISODate("2016-01-18T12:23:00.430Z"),
    "category": "call",
    "__v": 0
} {
    "_id": ObjectId("569cd88ba2f7c9e856268cb8"),
    "url": "/ivr/1453119757052",
    "method": "POST",
    "params": {
        "version": "1453119757052"
    },
    "licence_key": "e6053eb8d35e02ae40beeeacef203c1a",
    "app_id": "ubicall-admin",
    "status_code": 200,
    "status": "success",
    "datetime": ISODate("2016-01-18T12:20:27.040Z"),
    "category": "ivr",
    "__v": 0
} {
    "_id": ObjectId("569cd8a9a2f7c9e856268cb9"),
    "url": "/agent/calls",
    "method": "GET",
    "licence_key": "e6053eb8d35e02ae40beeeacef203c1a",
    "app_id": "ubicall-agent",
    "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.80 Safari/537.36",
    "status_code": 304,
    "status": "success",
    "datetime": ISODate("2016-01-18T12:20:57.774Z"),
    "category": "agent",
    "__v": 0
} {
    "_id": ObjectId("569cd8a9a2f7c9e856268cba"),
    "url": "/agent/queues",
    "method": "GET",
    "licence_key": "e6053eb8d35e02ae40beeeacef203c1a",
    "app_id": "ubicall-agent",
    "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.80 Safari/537.36",
    "status_code": 304,
    "status": "success",
    "datetime": ISODate("2016-01-18T12:20:57.996Z"),
    "category": "agent",
    "__v": 0
}
```
